src/logic/config/store/selectors/index.ts

```diff
@@ -13,6 +14,10 @@ export const currentChainName = createSelector([networkConfigState], (networkCon
   return networkConfig.chainName
 })
 
+export const currentNativeCurrency = createSelector([networkConfigState], (networkConfig): NativeCurrency => {
+  return networkConfig.nativeCurrency
+})
```

Comment:
Wow, the native coin getter was used in many places, wasn’t it.

I guess this new selector is supposed to supersede it? I wouldn’t add it if it’s not used anywhere yet, though. There’s something minimalistic about using the general confit getter + restructuring.